kvs/src/kvs/rep_factor_change_handler.cpp

```diff
@@ -118,17 +112,6 @@ void rep_factor_change_handler(
       } else {
         logger->error(
             "Missing key replication factor in rep factor change routine.");
-
-        // just update the replication factor
```

Comment:
Why remove this code?

Reply:
I was thinking that if the call to `get_responsible_thread` fails, then it will send `issue_replication_factor_request` and refresh the rep factor there...
This should actually never happen, so if it does we are screwed anyway...